source/settings/config-file.txt

```diff
@@ -126,6 +126,32 @@ Settings
        option but doesn't prevent users from manually enabling the
        option with the :guilabel:`Edit connection string` toggle. 
 
+   * - proxy
+     - Allows you to enforce or define a Proxy to connect to your deployment 
+       with. For more information on connecting to your deployment with a
+       proxy, see :ref:`ssh-connection`.
+
+       You must pass either a URL or a ``DevtoolsProxyOptions`` object as JSON
+       for the Proxy setting. A ``DevtoolsProxyOptions`` object takes the following
```

Comment:
There's a mixture of proxy with and without capital P here, might be good to align them (I see no reason why we'd capitalize it)